Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -431,6 +431,37 @@ void WritePixelShaderCommonHeader(ShaderCode& out, APIType ApiType, u32 num_texg
     }
   }
 
+  if (g_ActiveConfig.UseAccurateMipmaps()) {
```

Comment:
Our coding style puts the brace on the next line, ditto for all other occurrences.